test/e2e/framework/exec_util.go

```diff
@@ -58,6 +58,7 @@ func (f *Framework) ExecWithOptions(options ExecOptions) (string, string, error)
 
 	const tty = false
 
+	Logf("ExecWithOptions: Clientset creation ")
```

Comment:
do you still want to add this logging here?

Reply:
actually this is a good first issue i guess for someone to add so ill move it out of this PR https://github.com/kubernetes/kubernetes/issues/102295